src/pages/workspaces/workspace/notebooks/NotebookLauncher.js

```diff
@@ -253,6 +254,9 @@ const PreviewHeader = ({ queryParams, cluster, readOnlyAccess, onCreateCluster,
       [clusterStatus === 'Stopping', () => h(StatusMessage, [
         'Notebook runtime environment is stopping, which takes ~4 minutes. You can restart it after it finishes.'
       ])],
+      [clusterStatus === 'Reconfiguring', () => h(StatusMessage, [
```

Comment:
Same comment about `Updating` here as above.

Reply:
This one is fine, because `'Updating'` is in `usableStatuses`, so we should never get here with that status.

Reply:
Oh, that's interesting. I didn't expect that but I guess it makes sense. Maybe that's also the answer to the other case above? (Or at least evidence of the thinking behind why `Updating` isn't a case there.)

Reply:
Also, TOL: The `usableStatuses` logic influencing this `Utils.cond` is a little too far away for my liking. I wish the evidence that this isn't a bug were at least within this component, thus potentially avoiding this conversation. Since we are already having the conversation, I want to take advantage of the moment and learn from it. (Thank you @dmohs for enabling this kind of thinking for me!)

 I bring it up so that we think about this in the future, thus getting the most possible value out of this conversation now that it's happened. :)

Reply:
I endorse this line of thinking :)

Reply:
Elsewhere in this file we use `console.assert` to encode these assumptions explicitly, perhaps we could do that here as well.